ui/lib/sync/addon/components/secrets/sync-activation-modal.ts

```diff
@@ -43,7 +43,7 @@ export default class SyncActivationModal extends Component<Args> {
         .adapterFor('application')
         .ajax('/v1/sys/activation-flags/secrets-sync/activate', 'POST', { namespace });
       // must refresh and not transition because transition does not refresh the model from within a namespace
-      yield this.router.refresh();
+      yield this.router.refresh('vault.cluster');
```

Comment:
you can probably take this or leave it, but i tried explicitly passing in the parent route that we want refreshed here to see if it'd help us assert that the get activated features was successfully re-fetched. i tried this based on the docs i read here:
<img width="837" alt="Screenshot 2024-05-29 at 3 39 22 PM" src="https://github.com/hashicorp/vault/assets/903288/a0aefb66-1c69-45b4-94c6-06b7516e6608">

doing this works fine in the UI, and after adding a bunch of `console.log` statements i can see even in the test output that _`/activation-flags` _is_ re-fetched after a POST, but i can't get the assertions working how i'd like. because we have [these assertions](https://github.com/hashicorp/vault/blob/27df01adedfe12b696087d374e6b1bb3237e7ee4/ui/tests/acceptance/sync/secrets/overview-test.js#L128-L140) that effectively test that the UI updates once the feature is activated i think we're safe skipping the extra assertions.